src/analysisd/decoders/plugins/wazuhalert_decoder.c

```diff
@@ -40,11 +40,12 @@ void *OSSECAlert_Decoder_Exec(Eventinfo *lf, OSHash *rules_hash, __attribute__((
     char *tmp_str = NULL;
     RuleInfo *rule_pointer;
 
-    lf->decoder_info->type = OSSEC_ALERT;
+    lf->decoder_info->type = WAZUH_ALERT;
 
     /* Checking the alert level. */
     if (strncmp("Alert Level: ", lf->log, 12) != 0 &&
-       strncmp("ossec: Alert Level:", lf->log, 18) != 0) {
+        strncmp("ossec: Alert Level:", lf->log, 18) != 0 &&
```

Comment:
Same observation as in ruleset PR: I think these logs are only part of the manager and don't require this dual comparison.

Reply:
Updated!